README.md

```diff
@@ -32,69 +36,137 @@ Create `exams.sql` file with `INSERT INTO` query that adds entries to `exams` ta
 Write a query that outputs `year`, `modifier`, `student_cnt` columns that specify the number of students (`student_cnt`) in a class for all classes.
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT year || modifier AS class, 
+COUNT(*) AS students
+FROM class
+JOIN groups ON class.id = groups.class_id
+GROUP BY class_id
 ```
 
 ### 2. Find the number of students for every year
 
 Write a query that outputs `year`, `student_cnt` columns that specify the number of students (`student_cnt`) in a class for each year (`year`).
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT year, 
+COUNT(*) AS students
+FROM class
+JOIN groups ON class.id = groups.class_id
+GROUP BY year
 ```
 
 ### 3. Find unique subjects that students learn for a given year
 
 Write a query that outputs `name`, `surname`, `unique_lesson_cnt`, where `unique_lesson_cnt` is the number of **unique** lessons that a student identified by `name`, `surname` has.
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT name, surname,
```

Comment:
Grouping by name and surname is not correct. What if there are namesakes (i.e. students with the same name and surnames, but different IDs)?